docs/content/Checkbox.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Checkbox
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Checklist.tsx
+---
```

Comment:
We can also add a link to storybook here:
```suggestion
source: https://github.com/primer/react/blob/main/src/Checklist.tsx
storybook: ...
---
```

Reply:
👍 Added